packages/react-components/react-tag-picker-preview/cypress.config.ts

```diff
@@ -0,0 +1,3 @@
+import { baseConfig } from '@fluentui/scripts-cypress';
```

Comment:
please make sure to always use our generators for adding additional setup for packages to avoid issues like these https://github.com/microsoft/fluentui/pull/30989 which soon enough will fail PRs that will not comply 🥳

<img width="553" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/165ca9f6-6291-4295-9729-a3d242600266">